amqp/subscriber.go

```diff
@@ -28,7 +30,7 @@ type Subscriber interface {
 	SubscribeDeviceUplink(appID, devID string, handler UplinkHandler) error
 	SubscribeAppUplink(appID string, handler UplinkHandler) error
 	SubscribeUplink(handler UplinkHandler) error
-	ConsumeUplink(queue string, handler UplinkHandler) error
+	ConsumeMessages(queue string, uplinkHandler UplinkHandler, eventHandler DeviceEventHandler) error
```

Comment:
Yes. If they want the events they pass a handler and add bind the queue to receive events.